spec/models/passenger_spec.rb

```diff
@@ -35,4 +35,31 @@
       expect(passenger).not_to be_valid
     end
   end
+
+  describe 'rides_expire' do
+    context 'permaent passenger' do
+      it 'returns nil' do
+        @passenger.update permanent: true
+        expect(@passenger.rides_expire).to be nil
+      end
+    end
+    context 'doctors note present' do
+      it 'returns the expiration date of the note' do
+        date = 14.days.since.to_date
```

Comment:
Similarly, `Date.today + 14`

Reply:
Again I respectfully disagree that the plus sign is clearer. To me, it makes me look for math first. Although perhaps a discussion about why you think it's unclear is warranted.

Reply:
I'll change this to `from_now`

Reply:
There are two reasons I suggest the `+`:

1. `.since` is kind of confusing
2. using `14.days.since.to_date` you have to go from a `DateTime` to a `Date` for no real reason other than it looks subjectively better

Although looking at Matt's comment I agree that `.from_now` is much more readable.